The path of live os image in oe is hardcoded in anaconda-init.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/modules/payloads/source/live_os/live_os.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/pyanaconda/modules/payloads/source/live_os/live_os.py b/pyanaconda/modules/payloads/source/live_os/live_os.py
index 28a3767fe..2e38668dd 100644
--- a/pyanaconda/modules/payloads/source/live_os/live_os.py
+++ b/pyanaconda/modules/payloads/source/live_os/live_os.py
@@ -110,6 +110,9 @@ class LiveOSSourceModule(PayloadSourceBase, MountingSourceMixin):
         return [task]
 
     def detect_live_os_image(self):
+        # hardcode in anaconda-init
+        return '/dev/loop3'
+
         """Detect live os image in the system."""
         log.debug("Trying to detect live os base image automatically")
         for block_device in ["/dev/mapper/live-base", "/dev/mapper/live-osimg-min"]:
